__tests__/app.test.js

```diff
@@ -142,4 +142,69 @@ describe("/api/articles/:article_id", () => {
             expect(response.body.msg).toBe("Bad request")
         })
     })
+})
+
+describe("/api/articles/:article_id/comments", () => {
+    test("Get: 200 sends an array of comments belonging to a single article to the client", () => {
+        return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then((response) => {
+            const commentObj = {
+                comment_id: 9,
+                body: 'Superficially charming',
+                article_id: 1,
+                author: 'icellusedkars',
+                votes: 0,
+                created_at: '2020-01-01T03:08:00.000Z'
+              }
+            expect(response.body.comments[response.body.comments.length -1]).toMatchObject(commentObj)
```

Comment:
This is fine for now but not a very flexible test (the first part where you check for a specific comment), the rest of its good